demo/samples/background.cpp

```diff
@@ -49,20 +50,21 @@ void BackgroundDemo::onInit()
      */
     mFooter = new QGVWidgetText();
     geoMap()->addWidget(mFooter);
+
     /*
      * List of available tile maps.
      */
     const QString customURI = "http://c.tile.stamen.com/watercolor/${z}/${x}/${y}.jpg";
-    const QList<QPair<QString, QGVLayer*>> layers = {
-        { "OSM", new QGVLayerOSM() },
-        { "GOOGLE_SATELLITE", new QGVLayerGoogle(QGV::TilesType::Satellite) },
-        { "GOOGLE_HYBRID", new QGVLayerGoogle(QGV::TilesType::Hybrid) },
-        { "GOOGLE_SCHEMA", new QGVLayerGoogle(QGV::TilesType::Schema) },
-        { "BING_SATELLITE", new QGVLayerBing(QGV::TilesType::Satellite) },
-        { "BING_HYBRID", new QGVLayerBing(QGV::TilesType::Hybrid) },
-        { "BING_SCHEMA", new QGVLayerBing(QGV::TilesType::Schema) },
-        { "CUSTOM_OSM", new QGVLayerOSM(customURI) },
-    };
+    const QList<QPair<QString, QGVLayer*>> layers = { { "OSM", new QGVLayerOSM() },
+                                                      { "GOOGLE_SATELLITE",
+                                                        new QGVLayerGoogle(QGV::TilesType::Satellite) },
+                                                      { "GOOGLE_HYBRID", new QGVLayerGoogle(QGV::TilesType::Hybrid) },
+                                                      { "GOOGLE_SCHEMA", new QGVLayerGoogle(QGV::TilesType::Schema) },
+                                                      { "BING_SATELLITE", new QGVLayerBing(QGV::TilesType::Satellite) },
+                                                      { "BING_HYBRID", new QGVLayerBing(QGV::TilesType::Hybrid) },
+                                                      { "BING_SCHEMA", new QGVLayerBing(QGV::TilesType::Schema) },
+                                                      { "CUSTOM_OSM", new QGVLayerOSM(customURI) } };
```

Comment:
Please add colon for last item in init list - it looks better after clang-format